sdk/keyvault/Microsoft.Azure.KeyVault/tests/Microsoft.Azure.KeyVault.Tests.csproj

```diff
@@ -14,6 +14,7 @@
 
   <ItemGroup>
     <Compile Remove="TestFramework\**\*.cs" />
+    <Compile Remove="KeyVaultOperationsTest.cs" Condition="'$(OS)' != 'Windows_NT'" />
```

Comment:
Can you please transfer the comment over as well so it is still liked to the issue to fix this test?